tests/basilisp/reader_test.py

```diff
@@ -198,6 +198,32 @@ def test_reader_lines_from_str_other_loc(self, tmp_path):
             l3.meta.get(reader.READER_END_COL_KW),
         )
 
+    @pytest.mark.parametrize(
+        "evalstr,first,second",
+        [
+            ("[5]\n(def n 123)", (1, 1, 0, 3), (2, 2, 0, 11)),
```

Comment:
Can you add cases for buffers starting with `\n` and `\r\n` as well since we're looking back in the last 2 characters?

Reply:
Added the following, is this what you had in mind?

```python
            ("\n[5]\n(def n 123)", (2, 2, 0, 3), (3, 3, 0, 11)),
            ("\r[5]\r(def n 123)", (2, 2, 0, 3), (3, 3, 0, 11)),
            ("\r\n[5]\r\n(def n 123)", (2, 2, 0, 3), (3, 3, 0, 11)),
```

Reply:
Indeed. Thanks!